src/vmm/src/devices/virtio/block/vhost_user/device.rs

```diff
@@ -311,7 +311,7 @@ impl<T: VhostUserHandleBackend + Send + 'static> VirtioDevice for VhostUserBlock
         &self.queue_evts
     }
 
-    fn interrupt_trigger(&self) -> Arc<IrqTrigger> {
+    fn interrupt_trigger(&self) -> Arc<dyn VirtioInterrupt> {
```

Comment:
I suppose all of these could just return `&dyn VirtioInterrupt`, as none of the callers make use of the `Arc`s as far as I can see. But also out of scope here.

Reply:
Yup, shall we do this when we bring the MSI code in?